src/main/java/graphql/validation/schemawiring/ValidationSchemaWiring.java

```diff
@@ -34,15 +36,23 @@ public ValidationSchemaWiring(ValidationRules ruleCandidates) {
     public GraphQLFieldDefinition onField(SchemaDirectiveWiringEnvironment<GraphQLFieldDefinition> env) {
         GraphQLFieldsContainer fieldsContainer = env.getFieldsContainer();
         GraphQLFieldDefinition fieldDefinition = env.getFieldDefinition();
-
+        TargetedValidationRules rules = ruleCandidates.buildRulesFor(fieldDefinition, fieldsContainer);
+        if (rules.isEmpty()) {
```

Comment:
This should always have been present.  Dont build a new DataFetcher unless there is rules for that field

I am not sure how this got missed